src/libraries/src.proj

```diff
@@ -0,0 +1,38 @@
+<Project Sdk="Microsoft.Build.Traversal">
```

Comment:
It would've been nice if we used `git mv` for these files rename to see the diff and have the rename history in it.

Reply:
Yeah will do, @joperezr already mentioned that the last time but I again forgot for some files :(

Reply:
I tried everything. I jumped back into the initial commit where the rename was done and did a git mv back to .builds commited that and then again changed it via git mv to .proj but it still detects it as a deletion and creation GRRRRRR